webapp-django/crashstats/authentication/management/commands/makesuperuser.py

```diff
@@ -1,11 +1,19 @@
-import sys
+from __future__ import print_function
+
+try:
+    _input = raw_input
+except NameError:
+    # You're on Python >=3
+    _input = input
```

Comment:
We've got six in the requirements. I'd rather we used that since that's what it's for.